packages/yourloops/lib/team/team.hook.tsx

```diff
@@ -92,7 +92,15 @@ function TeamContextImpl(): TeamContext {
   }
 
   const getMedicalTeams = (): Team[] => {
-    return teams.filter((team: Team): boolean => team.type === TeamType.medical)
+    return getTeamsByType([TeamType.medical])
+  }
+
+  const getMedicalAndPrivateTeams = (): Team[] => {
```

Comment:
I did not find any UT in `packages/yourloops/test/unit/lib/team/team.hook.test.tsx` about these new methods, is this wanted?

Reply:
I completely forgot about them, it's fixed now!